core/store/migrate/migrations/0086_ocr2_tables.sql

```diff
@@ -0,0 +1,182 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE offchainreporting2_oracle_specs (
+     id SERIAL PRIMARY KEY,
+     contract_address bytea NOT NULL,
+     p2p_peer_id text,
+     p2p_bootstrap_peers text[],
```

Comment:
note that does mean we need set the P2PBootstrapPeers to non-nil since its a pq.StringArray, but that looks ok since we can populate it in the validate function